LocalSettings.php

```diff
@@ -2115,6 +2114,36 @@
 			'bureaucrat' => [
 				'bureaucrat' => true,
 			],
+			'extendedconfirmed' => [
+				'extendedconfirmed' => true,
+			],
+			'leader' => [
+				'leader' => true,
+			],
+			'steward' => [
```

Comment:
Just wondering what you're doing modifying the steward usergroup locally? Seem to be adding oversight and CheckUser into Steward. I'm not certain, but that should be discouraged. Stewards can add themselves to those individual usergroups when really necessary.

Reply:
Well, I will just end up not doing it and remove that part of the changes, but I will keep the rest of the changes.

Reply:
I have now updated it so that's no longer happening.